Index: event/EventTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package event;\r\n\r\nimport org.junit.jupiter.api.Test;\r\n\r\n\r\n/**\r\n * A Unit Test Class for Event.java.\r\n */\r\nclass EventTest {\r\n    /**\r\n     * A Unit Test for Event.complete().\r\n     */\r\n    @Test\r\n    void complete() {\r\n\r\n    }\r\n}
===================================================================
diff --git a/event/EventTest.java b/event/EventTest.java
--- a/event/EventTest.java	
+++ b/event/EventTest.java	
@@ -1,6 +1,11 @@
 package event;
 
 import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDateTime;
+
+import timeBehaviour.TimePoint;
 
 
 /**
@@ -11,7 +16,15 @@
      * A Unit Test for Event.complete().
      */
     @Test
-    void complete() {
+    void completeTest() {
+        Event e = new Event("Event1", "testing", 100, new TimePoint(LocalDateTime.now()));
+
+        assertEquals(0, Event.getObserverList().size());
+        assert(Event.getObserverList().contains(e));
 
+        e.complete();
+
+        assertEquals(0, Event.getObserverList().size());
+        assertFalse(Event.getObserverList().contains(e));
     }
 }
\ No newline at end of file
